ndcube/mixins/ndslicing.py

```diff
@@ -49,7 +49,10 @@ def _slice(self, item):
         if temp_extra_coords is not None:
             kwargs['extra_coords'] = list(temp_extra_coords) + list(dropped_coords)
         else: 
-            kwargs['extra_coords'] = list(dropped_coords)
+            if len(dropped_coords) != 0:
+                kwargs['extra_coords'] = list(dropped_coords)
+            else:
+                kwargs['extra_coords'] is None
```

Comment:
This should be ```kwargs['extra_coords'] = None```.  What you've written will evaluate whether ```kwargs['extra_coords']``` is ```None``` and return a True or False.

Reply:
Thanks for the correction!